Move the cloud output to the new Output interface

na-- opened the pull request:
Follow up to https://github.com/loadimpact/k6/pull/1869 and https://github.com/loadimpact/k6/pull/1870. Closes https://github.com/loadimpact/k6/issues/1258 and moves the cloud output to the new `Output` interface.

MStoykov commented on cloudapi/util.go:
```diff
@@ -22,10 +22,7 @@ package cloudapi
-		path = "anonymous"
-	}
-	return fmt.Sprintf("%s/%s/%s", config.WebAppURL.String, path, refID)
+	return fmt.Sprintf("%s/runs/%s", config.WebAppURL.String, refID)
```
nit ... maybe just concat them with `+` ...?

na-- replied:
resolved by https://github.com/loadimpact/k6/compare/a67b3a127823480eff7038aa32f812d93ad5a2ac..7d50a9ea51f50006cad35a246b29ac1623ad8699

MStoykov reviewed (approved):
LGTM.

MStoykov reviewed (changes requested):
actually just checked 2 or 3 things I wasn't sure about with the config and a thing that definitely doesn't work is picking up my subscription from the config file

na-- commented:
This is now ready for merging from my side, I've renamed all `Collector` / `c` mentions to `Output` / `out`, and I've cleaned up some of the linter issues that were present. I suggest reading the changes commit by commit.
